### PR TITLE
chore(deps): bump launchdarkly-java-server sdk to version 5.0.0

### PR DESCRIPTION
**Pull request by Gogetter**
### JIRA link (if applicable) ###
NA

### Change description ###
Following [launchdarkly updates](https://docs.launchdarkly.com/sdk/server-side/java/migration-4-to-5) dependabot update #1302 failed.

This PR adds a few more resolutions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/build.gradle [hunk 1]
```diff
@@ -194,9 +194,8 @@ dependencies {
   compile group: 'org.springframework.security', name: 'spring-security-oauth2-resource-server'
   compile group: 'org.springframework.security', name: 'spring-security-oauth2-jose'
   compile group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '8.17.1' // override to fix vulnerability issue (CVE-2019-17195)
-  compile group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.7.2'
-  compile group: 'com.launchdarkly', name: 'okhttp-eventsource', version: '1.11.0'
```
Comment (Gogetter) [inline service/build.gradle:198]: removed both as we have now taken `classifier` from `launchdarkly-java-server-sdk`

File: service/build.gradle [hunk 2]
```diff
@@ -194,9 +194,8 @@ dependencies {
...
-  compile group: 'com.launchdarkly', name: 'launchdarkly-java-server-sdk', version: '4.14.0', classifier: 'thin'
```
Comment (Gogetter) [inline service/build.gradle:199]: removed `classifier: thin` so we are not forced to use `gson`. See [here](https://docs.launchdarkly.com/sdk/server-side/java/migration-4-to-5#understanding-changes-to-the-json-value-type)

File: service/src/main/java/uk/gov/hmcts/reform/fpl/config/scheduler/JobFactory.java [hunk 3]
```diff
@@ -1,7 +1,6 @@
 package uk.gov.hmcts.reform.fpl.config.scheduler;
 
 import lombok.RequiredArgsConstructor;
-import org.jetbrains.annotations.NotNull;
```
Comment (Gogetter) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/config/scheduler/JobFactory.java:4]: looks like this was a transitive dep from `com.squareup.okhttp3`

File: service/build.gradle (hunk 2 above)
Comment (akomisarek) [reply service/build.gradle:215]: I am not sure if I understood your rationing correctly, I don't see how `thin` is "forcing" us to use `gson`. Please see [here]https://docs.launchdarkly.com/sdk/server-side/java/migration-4-to-5#using-other-third-party-dependencies)
Now Gson is simply embedded in the library and we potentially use different version, but I think I am happy with it, if it works I don't see reason to change especially if we can remove one of our dependences.

**Pull request merged**